circuit-prover/src/air/witness_air.rs

```diff
@@ -1,26 +1,41 @@
-//! [`WitnessAir`] is the central AIR in our construction of a recursive verifier. It represents the **global witness bus**,
-//! storing all witness indices and associated values used within the verifier circuit.
-//! The generic parameter `D` allows the AIR to handle values from an extension field of degree `D` over the base field.
+//! [`WitnessAir`] defines the AIR for the global witness bus used by every other circuit table.
```

Comment:
nit:
```suggestion
//! [`WitnessAir`] defines the AIR for the global witness bus used by all other circuit tables.
```
(might be just me, but it sounded to me like it was half the tables with "every other table")